jhu/delphi_jhu/run.py

```diff
@@ -86,6 +87,12 @@ def run_module():
         METRICS, GEO_RESOLUTIONS, SENSORS, SMOOTHERS
     ):
         print(metric, geo_res, sensor, smoother)
+        logger.info(
+            event="processing data",
```

Comment:
is this descriptive enough (e.g. isn't everything in this pipeline "processing data")? I guess the logs will have file/indicator name and the metric/geo/sensor/smoother which seems sufficient, but haven't followed the logging so don't know how these all get used/parsed.

Reply:
I updated to be a bit more specific here, but yeah we should have lots of information to use in the logs. We'll likely go back and edit these as we start understanding what it most helpful to collect.